assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/multiuser.properties

```diff
@@ -40,6 +40,10 @@ che.limits.workspace.env.ram=16gb
 # counts toward idleness.
 che.limits.workspace.idle.timeout=1800000
 
+# The length of time that a workspace will run, regardless of activity, before
+# the system will suspend it.
```

Comment:
IMO it worth clarifying a couple of things:
- 0 = disabled by default
- what time unit is expected to be used in the property (milliseconds ? similar to what is used in the `che.limits.workspace.idle.timeout` )

Reply:
the proper detailed explanation of the property is important since it will automatically generate the docs - https://www.eclipse.org/che/docs/che-7/advanced-configuration-options-for-the-che-server-component/#che-server-component-system-properties-reference_advanced-configuration-options-for-the-che-server-component

Reply:
Updated the documentation to clarify these points